common.blocks/i-bem-dom/i-bem-dom.js

```diff
@@ -905,9 +914,8 @@ bemDom = /** @exports */{
         });
 
         // NOTE: it was moved here as jquery events aren't triggered on detached DOM elements
-        excludeSelf?
-            ctx.empty() :
-            ctx.remove();
+        _destructDom &&
+            (excludeSelf? ctx.empty() : ctx.remove());
```

Comment:
@dfilatov do you remember why can't we do similar things in `v2` in the same way?